bin/start-kind.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
```

Comment:
I will @rouja review this file.

Reply:
/bin/sh is an executable representing the system shell and usually implemented as a symbolic link pointing to the executable for whichever shell is the system shell. The system shell is basically the default shell that the script should use. In last couple of years, Debian (and Ubuntu) decided to switch the system shell from bash to dash — a similar shell but lighter and much faster.